Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1000,19 +1036,33 @@ def search_indicator(indicator_type, indicator_value):
         'includeTags': 'true',
     }
 
+    demisto.debug(f'search_indicator: {indicator_value=}')
+    result = Response()
     try:
-        result = requests.request(
-            method='GET',
-            url=f'{BASE_URL}/tic',
-            verify=USE_SSL,
-            headers=headers,
-            params=params
-        )
+        if client:
+            demisto.debug('search_indicator: using the client')
+            # 404, 409, 503 a list of status codes that we want to handle in the code and not raise automatic exception about.
+            ok_codes = (200, 404, 409, 503)
+            result = client.get_url_enrichment(headers, params, ok_codes)
+        else:
+            result = requests.request(
+                method='GET',
+                url=f'{BASE_URL}/tic',
+                verify=USE_SSL,
+                headers=headers,
+                params=params
+            )
+        demisto.debug(f'search_indicator: {result.status_code=} {result=}')
 
-        result_json = result.json()
+        try:
+            result_json = result.json()
+        except ValueError:
+            result_json = {}
```

Comment:
Maybe add a debug message?

Reply:
There is a debug message in the next line.